### PR TITLE
Feature/subscription line item updates

### PR DESCRIPTION
**Pull request by LukeSettle**
Adds in serializers for cfsm to replace `to_json` method that was used in multiple places.

### CODE REVIEW

**Review by cnorm35 — APPROVED** [review]
This looks great.  I'm a big fan of AMS but we should probably start looking into better alternatives based on their current status:

https://github.com/rails-api/active_model_serializers#status

it's been 2yrs since the last non-readme update

**Comment by LukeSettle** [comment]
@cnorm35 Oh wow. I didn't even look at their repo when I added it... I always use AMS 😢

**Review by jacobeubanks — APPROVED** [review]

**Review by jacobherrington — APPROVED** [review]
👍

File: app/controllers/solidus_subscriptions/api/v1/subscriptions_controller.rb [hunk 1]
```diff
@@ -4,7 +4,11 @@ class SolidusSubscriptions::Api::V1::SubscriptionsController < Spree::Api::BaseC
   def update
     if @subscription.update(subscription_params)
       persist_subscription_addresses(@subscription)
-      render json: @subscription.to_json(include: [:line_items, :shipping_address, :billing_address, :wallet_payment_source])
+      if params[:full_json]
+        render json: SubscriptionSerializer.new(@subscription).to_json(include: '**')
+      else
+        render json: @subscription.to_json(include: [:wallet_payment_source, :shipping_address, :billing_address])
```
Comment (fastjames) [inline app/controllers/solidus_subscriptions/api/v1/subscriptions_controller.rb]: I note that this is a small behavior change from the old code, leaving out `:line_items` in the fallback case here.

**Review by fastjames — APPROVED** [review]
:cake:

**Pull request merged**